### ISSUE TITLE
Use a single BindAddress setting for API & webhooks

### ISSUE THREAD

**Issue by patcon** [42wim/matterbridge#555]
It seems that we're setting bindaddress differently in different places: 
- WebhookBindAddress for Slack/Mattermost/Rocketchat
- BindAddress for API

Would it not make more sense to go back to using BindAddress generally? It seems like a more rare case where admins would want to individually set BindAddress to different ports, no?

In other words, let's set BindAddress in General for a default, and if BindAddress is set on each protocol, override for the webhook.

This will also make it simpler to set a `-bindaddress <addr>` flag from the command-line, which I'm hoping to add soon

**Comment by 42wim** [comment]
No, because it's a Webhook Bindaddress for slack/mattermost/rocketchat.
The api isn't just for webhooks.

Also I'm not for adding extra configuration via command-line. It can be done via ENV and via configuration files. That should be enough.

**Comment by patcon** [comment]
Actually, that works for me, re envvars :)

Respectfull, I'm still not sure I understand why slack/mattermost/rocketchat would need their own bindaddress config. Are you imagining that they're listening on different ports? And if not, would it be an option to just have a toggle to enable/disable each one, but have one bindaddress, instead of letting the bindaddress setting imply activation?

**Comment by patcon** [comment]
Fwiw "Imagining" was just the word I used when I thought it was a possible config, instead of the defacto way :)
